_posts/2019-11-06-hhvm-4.30.markdown

```diff
@@ -0,0 +1,57 @@
+---
+title: "HHVM 4.30.0"
+layout: post
+author: jjergus
+category: blog
+---
+
+HHVM 4.30 is released! This release marks the end of support for 4.24;
+4.25&ndash;4.29 remain supported, as do the LTS releases 3.30 and 4.8.
+
+# Highlights
+
+- `hh_client` supports a new command `--extract-standalone` which extracts the
```

Comment:
- not sure if this is ready to announce
- I /think/ it stubs out dependencies rather than extracting their implementations. This probably needs clarifying; if this is the case, it's primarily useful for typechecker/parser tests, not anything involving runtime

Reply:
In particular, it can't be used as an alternative to `phar`s